src/Microsoft.ML.AutoML/API/MulticlassClassificationExperiment.cs

```diff
@@ -136,7 +141,8 @@ internal MulticlassClassificationExperiment(MLContext context, MulticlassExperim
         public override ExperimentResult<MulticlassClassificationMetrics> Execute(IDataView trainData, ColumnInformation columnInformation, IEstimator<ITransformer> preFeaturizer = null, IProgress<RunDetail<MulticlassClassificationMetrics>> progressHandler = null)
         {
             var label = columnInformation.LabelColumnName;
-            _experiment.SetEvaluateMetric(Settings.OptimizingMetric, label);
+            TrialResultMonitor<MulticlassClassificationMetrics> monitor = null;
```

Comment:
>TrialResultMonitor<MulticlassClassificationMetrics> monitor = null; [](http://example.com/codeflow?start=12&length=67)

nit: maybe better move this line down before _experiment.SetMonitor line?

Reply:
### Resolved